docs/monitoring.md

```diff
@@ -220,6 +220,13 @@ The history server can be configured as follows:
       Number of threads that will be used by history server to process event logs.
     </td>
   </tr>
+  <tr>
+    <td>spark.history.store.path</td>
```

Comment:
I'm using this as a trigger of whether to use the disk store or not; if you set a local store directory, you're using the disk store, otherwise you're using the memory store. I do need to update it in the documentation, though.